kyo-prelude/shared/src/main/scala/kyo/kernel/Flat.scala

```diff
@@ -75,9 +76,10 @@ private object FlatMacro:
                 case AppliedType(base, _) =>
                     base =:= TypeRepr.of[Maybe] ||
                     base =:= TypeRepr.of[Result] ||
-                    base =:= TypeRepr.of[TypeMap] ||
-                    base =:= TypeRepr.of[Duration]
```

Comment:
`Duration` was incorrect here since it doesn't have type parameters and isn't an `AppliedType`